plugins/poll/plugin.rb

```diff
@@ -486,6 +486,7 @@ def groupable_user_fields
   end
 
   validate(:post, :validate_polls) do |force = nil|
+    self.extracted_polls = []
```

Comment:
this feels a bit weird. a side effect of calling validations is setting / resetting a var to []. 

Is there any other way we can do this fix? More nil checks elsewhere?